corehq/apps/app_manager/forms.py

```diff
@@ -63,3 +64,76 @@ def clean(self):
         if self.cleaned_data.get('linked'):
             if not LINKED_APPS.enabled(domain):
                 raise forms.ValidationError("The target project space does not have linked apps enabled.")
+
+
+class PromptUpdateSettingsForm(forms.Form):
+    app_prompt = forms.ChoiceField(
+        label=ugettext_lazy("Prompt Updates to Latest Released App Version"),
+        choices=(
+            ('off', ugettext_lazy('Off')),
+            ('on', ugettext_lazy('On')),
+            ('forced', ugettext_lazy('Forced')),
+        ),
+        help_text=ugettext_lazy(
+            "If enabled, users will receive in-app prompts to update "
+            "to the latest released version of the app, if they are not "
+            "already on it. (Selecting 'Forced' will make it so that users "
+            "cannot continue to use CommCare until they update)"
+        )
+    )
+
+    apk_prompt = forms.ChoiceField(
+        label=ugettext_lazy("Prompt Updates to Latest CommCare Version"),
+        choices=(
+            ('off', ugettext_lazy('Off')),
+            ('on', ugettext_lazy('On')),
+            ('forced', ugettext_lazy('Forced')),
+        ),
+        help_text=ugettext_lazy(
+            "If enabled, users will receive in-app prompts to update "
+            "to the latest version of CommCare, if they are not already "
+            "on it. Selecting 'Forced' will make it so that users cannot "
```

Comment:
@sravfeyn Can you place the `"Selecting 'Forced' ... until they upgrade"` portion of this text in parentheses so that it matches the text above.